int32 cameraExp
int32 videoMode
